diff --git a/node_modules/string-width-cjs/index.js b/node_modules/string-width-cjs/index.js
index f4d261a..6d25b31 100644
--- a/node_modules/string-width-cjs/index.js
+++ b/node_modules/string-width-cjs/index.js
@@ -1,7 +1,7 @@
 'use strict';
-const stripAnsi = require('strip-ansi');
-const isFullwidthCodePoint = require('is-fullwidth-code-point');
-const emojiRegex = require('emoji-regex');
+import stripAnsi from 'strip-ansi'
+import isFullwidthCodePoint from 'is-fullwidth-code-point'
+import emojiRegex  from 'emoji-regex'
 
 const stringWidth = string => {
 	if (typeof string !== 'string' || string.length === 0) {
